src/components/Button.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-function Button({ name }) {
+function Button({ name, color, wide }) {
+  const style = {
+    backgroundColor: color,
+    width: wide ? '50%' : '25%',
+  };
+
   return (
-    <button type="submit">{ name }</button>
+    <button type="button" className="button" style={style}>{ name }</button>
```

Comment:
Let's avoid using inline styling, we could use the orange prop to make reference to a class instead, that way we can avoid the inline styles.